codes/quantum/qubits/lifted_product.yml

```diff
@@ -7,30 +7,35 @@ code_id: lifted_product
 physical: qubits
 logical: qubits
 
-name: 'Lifted product code'
-introduced: '\cite{arXiv:1904.02703}'
+name: Lifted product code
 
-description: 'A family of \([[n,k,d]]\) CSS codes whose construction is based on two linear \textit{seed} codes \(C_1\) and \(C_2\) with a free action of a group \(G\). Lifted product codes can also be defined over arbitrary matrix rings.'
+introduced: \cite{arxiv:1904.02703}
 
-protection: 'Code performance strongly depends on \(G\), but lifted product codes contain \cite{arXiv:2111.03654} families of QLDPC codes with the parameters \([[n, k = \Theta(n), d = \Theta(n)]]\).'
+description: >
+  A quantum LDPC code that is the "lifted version" of a hypergraph product code, formerly known as "generalized hypergraph product codes" \cite{arxiv:1904.02703}, and later renamed to "lifted product codes" \cite{arxiv:2012.04068,doi:10.1103/PRXQuantum.2.040101}.
```

Comment:
generalized hypergraph product statement should be moved into notes field to avoid confusion.